src/app.js

```diff
@@ -38,8 +37,8 @@ export default function (app) {
     }
 
     if (obj = plugin.hooks) {
-      Object.keys(obj).forEach(function (key) {
-        hooks[key].push(obj[key])
+      Object.keys(obj).map(function (key) {
```

Comment:
In case anyone is wondering: Using `map` here makes no important difference as opposed to using forEach and we can save more bytes.